### PR TITLE
Adds a page with registration information.

### PR DESCRIPTION
**Pull request by Robadob**
Interested in whether you think there's anything more I should add @ns-rse.

I also still need to get the feedback form off you to add to the end of the course.

My idea with the registration page is to add a link to the main RSE website nav that links to pando.

### CODE REVIEW

File: learners/registration.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: 'Registration Information'
+---
+
+*If you're external to the University of Sheffield and are interested in us delivering the course at your institution, delivering or taking it yourself, [reach out](mailto:robert.chisholm@sheffield.ac.uk?subject=[pando-external]) for more information or advice.*
+
+This course currently runs for PhD students and staff at the University of Sheffield roughly once a month on Thursdays.
+
+If you're interested in registering to take the course, and have checked that you meet the [prerequisites](/index.html#prerequisites) (this course isn't very suitable for Python beginners), search for "profiling" on [MyDevelopment](https://www.sheffield.ac.uk/nap/service/redirect/mydev) and sign up for the next run that fits your schedule.
```
Comment (ns-rse) [inline learners/registration.md:9]: I'd remove the `very` just to make it quite clear that beginners shouldn't be taking this course.

```suggestion
If you're interested in registering to take the course, and have checked that you meet the [prerequisites](/index.html#prerequisites) (this course isn't suitable for Python beginners), search for "profiling" on [MyDevelopment](https://www.sheffield.ac.uk/nap/service/redirect/mydev) and sign up for the next run that fits your schedule.
```

**Review by ns-rse — COMMENTED** [review]
Nothing significant to add, although I'd be somewhat wary of making the Workbench too specific to Sheffield.

This is the sort of content/material I think would sit nicely as a separate page linked from [RSE Sheffield - Training](https://rse.shef.ac.uk/training/).

**Comment by Robadob** [comment]
I'm fine with making it Sheffield specific. When I submit to carpentry
incubator it will be a fork with Sheffield stripped out.

**Comment by Robadob** [comment]
I'll adjust this with direct link to MyDevelopment and add a comment saying you may need to log into mydevelopment before clicking this link 🙄, before I merge. But otherwise ready to go.

**Review by ns-rse — APPROVED** [review]
:cool:

**Pull request merged**